openlibrary/catalog/marc/find_translation.py

```diff
@@ -10,7 +10,7 @@
     line = get_first_tag(data, set(['500']))
     if not line:
         continue
-    subtag, value = get_all_subfields(line).next()
+    subtag, value = next(get_all_subfields(line))
```

Comment:
This whole `openlibrary/catalog/marc/find_translation.py` script can be deleted, it has a Python _2.5_ shebang, and looks like a CLI output only script, which uses the deprecated code in marc.fast_parse. 

The other changes look good :+1: , I'll accept and merge when this file is deleted.